media/static-bundles.json

```diff
@@ -1327,12 +1327,8 @@
     },
     {
       "files": [
-        "js/libs/jquery.waypoints.min.js",
```

Comment:
Removing these two files is throwing an error `TypeError: Waypoint is undefined`

Reply:
Oh darn, I didn't realize we were using the waypoints for two different things. I got excited thinking I could remove the extra jQuery when the features section was removed.